tests/fixtures/migration-guard/profiles_without_signup_trigger.sql

```
@@ -0,0 +1,18 @@
+-- The table exists, the policies are right, the grants are right. The only thing
+-- missing is a writer, and the app finds out about it at the checkout route,
+-- which cannot attach a Stripe customer to a row that is not there.
+
+create table if not exists public.profiles (
+  id uuid primary key references auth.users(id) on delete cascade,
+  tier text not null default 'free'
+);
+
+alter table public.profiles enable row level security;
+
+create policy "profiles: select own"
```

SQLFluff (4.2.2)

[error] Do not use special characters in identifiers.

(RF05)